pinot-controller/src/main/resources/app/utils/PinotMethodUtils.ts

```diff
@@ -209,7 +209,11 @@ const getTableSchemaData = (tableName) => {
 
 const getAsObject = (str: SQLResult) => {
   if (typeof str === 'string' || str instanceof String) {
-    return JSON.parse(JSON.stringify(str));
+    try {
+      return JSONbig.parse(str);
+    } catch(e) {
+      return JSON.parse(JSON.stringify(str));
+    }
```

Comment:
I do wonder if this is even necessary now. JSON.parse is nice because it returns the string when it fails But I wonder if we should just swallow the exception and return the string below. I do think I prefer it like this since there's no test coverage, and I'm not sure what the expectation was here.

Reply:
Yes, since we don't have test coverage, it is better to handle it with catch here.